src/vs/platform/windows/common/windows.ts

```diff
@@ -265,6 +265,11 @@ export function getTitleBarStyle(configurationService: IConfigurationService, en
 		return 'native'; // not enabled when developing due to https://github.com/electron/electron/issues/3647
 	}
 
+	const accessibilityMode = configurationService.getValue('editor.accessibilitySupport');
+	if (!isMacintosh && accessibilityMode) {
```

Comment:
@sbatten the value of `editor.accessibilitySupport` is a string, so how does your check here actually work?